### DISCUSSION TITLE
请教规则写法

### DISCUSSION THREAD

**Discussion by jiuqianyuan** [gkd-kit/gkd#466] [Q&A]
在没有quickFind参与下，参考[README文档](https://github.com/gkd-kit/selector/blob/0591bbc03503865b2fee0a5f3e54041b050cc876/README.md?plain=1#L125-L141)，也是前者写法效率更高吗

`FrameLayout > FrameLayout[index=1] > @View[index=2][clickable=true] + TextView`
`TextView - @View[clickable=true] <3 FrameLayout <2 FrameLayout`
https://i.gkd.li/import/13832104

![无标题](https://github.com/gkd-kit/gkd/assets/39406781/37bf9ffc-194b-42b6-86b4-e91ec307c7f8)

**Comment by huowutong** [comment]
两者之间的耗时可以忽略不计吧，除非嵌套的层级到达一定的数量，更多的是写法和代码量的区别。

**Comment by lisonge** [comment]
效率最高的是 不使用关系选择器，此外全部使用 quickFind

就目前的选择器版本而言，如果一定要使用关系选择器，尽量不要使用 +/- 这两个选择器，此外使用精确距离，不要使用 n
